src/itest/java/uk/gov/companieshouse/pscdataapi/steps/PscDataSteps.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.companieshouse.pscdataapi.steps;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.assertj.core.api.Assertions;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.companieshouse.pscdataapi.config.AbstractMongoConfig.mongoDBContainer;
+
+import uk.gov.companieshouse.pscdataapi.config.CucumberContext;
+import uk.gov.companieshouse.pscdataapi.util.FileReaderUtil;
+import uk.gov.companieshouse.pscdataapi.repository.CompanyPscRepository;
+
+import java.io.IOException;
+import java.util.Collections;
+
+public class PscDataSteps {
+    private String contextId;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+    @Autowired
+    private MongoTemplate mongoTemplate;
+
+    @Autowired
+    private CompanyPscRepository companyPscRepository;
+
+    private final String companyNumber = "34777772";
+
+    @Before
+    public void dbCleanUp(){
+        if (!mongoDBContainer.isRunning()) {
+            mongoDBContainer.start();
+        }
+        companyPscRepository.deleteAll();
+    }
+
+    @Given("Psc data api service is running")
+    public void theApplicationRunning() {
+        assertThat(restTemplate).isNotNull();
+    }
+
+    @When("I send a PUT request with payload {string} file for company number with notification id {string}")
```

Comment:
You don't need to mention company number here
```suggestion
    @When("I send a PUT request with payload {string} file with notification id {string}")
```